observer/obs_conf.go

```diff
@@ -56,12 +57,33 @@ func (c *ObsConf) validateDebugAddr() error {
 	return nil
 }
 
-func (c *ObsConf) makeMonitors() ([]*monitor, []error, error) {
+func (c *ObsConf) makeMonitors(reg *prometheus.Registerer) ([]*monitor, []error, error) {
```

Comment:
When `makeMonitors()` is called in `obs_conf.go` this var is called `metrics`, it makes the code easier to read if we keep the same name here as well.